### PR TITLE
feat: migrated python examples to http2

### PR DESCRIPTION
**Pull request by dchebakov**

### CODE REVIEW

File: python/recognize_rest.py [hunk 1]
```diff
@@ -1,31 +1,34 @@
...
 
     if args.encoding == stt_pb2.RAW_OPUS:
         raise ValueError("RAW_OPUS encoding is not supported by this script")
     with audio_open_read(args.audio_file, args.encoding, args.rate, args.num_channels, args.chunk_size,
                          args.pyaudio_max_seconds) as reader:
-        metadata = authorization_metadata(args.api_key, args.secret_key, "tinkoff.cloud.stt", type=dict)
-        request = build_recognition_request(args, reader, type="json")
-        response = requests.post("http{}://{}/v1/stt:recognize".format("s" if args.endpoint.endswith("443") else "",
+
+        async with httpx.AsyncClient(http2=True) as client:
```
Comment (fsatka) [inline python/recognize_rest.py:19]: i would do it like another example

Comment (dchebakov) [reply python/recognize_rest.py:19]: first time I also thought to accomplish it as a separate example, but I. Lapshenkov said is not to show target users examples which working bases on http1 (the requests module in python doesn't support http2), because we are going to have a dedicated proxy for http1

by the way, if you will try to synthesize really long text using rest and http1, our balancer will return an internal error, but if you switch to http2, everything will be fine

**Review by dilap54 — APPROVED** [review]

File: python/snippets/tts_synthesize_linear16_playback.py [hunk 2]
```diff
@@ -31,7 +31,19 @@ def build_request():
         ),
     )
 
-stub = tts_pb2_grpc.TextToSpeechStub(grpc.secure_channel(endpoint, grpc.ssl_channel_credentials()))
+# increase this number if you are trying to synthesize really long text
+ONE_HUNDRED_MEGABAIT_BUFFER = 100 * 1024 * 1024
```
Comment (mykola-semenov) [inline python/snippets/tts_synthesize_linear16_playback.py]: MEGABYTE

File: python/README.md [hunk 3]
```diff
@@ -57,7 +57,7 @@ Disable:
 $ python3 recognize.py -r 16000 -c 1 -e MPEG_AUDIO --disable_profanity_filter ../audio/sample_5.mp3
 ```
 
-Also there is a REST-like API for non-streaming speech recognition:
+Also, there is a REST-like API for non-streaming speech recognition (if you are planning to use REST in production, which is not recommended, by the way, make sure to use libraries which support HTTP2, like httpx):
```
Comment (mykola-semenov) [inline python/README.md]: Is this relevant to the ListVoices method?

Comment (dchebakov) [reply python/README.md]: I think in general it would be better to use http2 everywhere

File: python/snippets/tts_synthesize_linear16_playback.py (hunk 2 above)
Comment (dchebakov) [reply python/snippets/tts_synthesize_linear16_playback.py]: fixed

**Review by mykola-semenov — APPROVED** [review]
